server/server.ts

```diff
@@ -141,6 +142,7 @@ export default async function (
 	}
 
 	if (!Config.values.https.enable) {
+		// @ts-ignore
```

Comment:
... you get the gist by now I think :smile: 
I'll stop dumping those for the cases further down the list, but all of them have the same issue